GoogleMapsComponents/GoogleMap.razor

```diff
@@ -79,5 +79,6 @@
         //    foreach (var mi in moduleImports)
         //        await mi.DisposeAsync();
         //}
+        base.Dispose();
```

Comment:
Not sure why we are not disposing here? I added this to make sure that we can clear up eventlistener and resources that belong to the map. There might be a good reason that this is not here?